Prox/Prox/Widgets/ReviewContainerView.swift

```diff
@@ -5,8 +5,16 @@
 import UIKit
 import QuartzCore
 
+enum ReviewContainerViewMode {
```

Comment:
see comment earlier about encapsulating specific configurations inside the `UIViewController` rather than inside the `ReviewContainerView`. This enum should be removed and the constants below turned into variables that can be configured externally.